### PR TITLE
book updates (min viable changes prior to release)

### PR DESCRIPTION
**Pull request by unixpi**

### CODE REVIEW

File: docs/the_nimbus_book/src/keymanager-api.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
 # Keymanager API
 
-The standardized [Keymanager API](https://ethereum.github.io/keymanager-APIs/) can be used to add, remove or migrate validators on the fly on a running beacon node.
+> ⚠️ This feature is currently in BETA - we are still testing it and implementation details may change in response to community feedback. **We strongly advise against using it on mainnet** for the time being
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/keymanager-api.md]:
```suggestion
> ⚠️ This feature is currently in BETA - we are still testing it and implementation details may change in response to community feedback. **We strongly advise against using it on mainnet** - your validators may get slashed
```

**Review by arnetheduck — APPROVED** [review]

File: docs/the_nimbus_book/src/keymanager-api.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+```
+./run-prater-beacon-node.sh --keymanager
+```
 
-> **Warning:** The Keymanager API port SHOULD be exposed through a secure channel, such as with HTTPs, an SSH tunnel, a VPN, etc.
+Once the node is running, you'll be able to access the API from [http://localhost:5052/](http://localhost:5052/)
 
-## Specification
+### Authorization: Bearer scheme
+All requests must be authorized through the `Authorization: Bearer` scheme. with a token matching the contents of a file provided at the start of the node through the `--keymanager-token-file` parameter.
```
Comment (zah) [inline docs/the_nimbus_book/src/keymanager-api.md]: Why is there a full stop after `scheme`?

Comment (unixpi) [reply docs/the_nimbus_book/src/keymanager-api.md]: typo. thanks

Comment (unixpi) [reply docs/the_nimbus_book/src/keymanager-api.md]:
```suggestion
All requests must be authorized through the `Authorization: Bearer` scheme with a token matching the contents of a file provided at the start of the node through the `--keymanager-token-file` parameter.
```

**Pull request merged**